packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -86,12 +101,25 @@ class AnnounceSemanticsEvent extends SemanticsEvent {
   /// This property must not be null.
   final TextDirection textDirection;
 
+  /// Determines whether the announcement should interrupt any existing announcement,
+  /// or queue after it.
+  ///
+  /// On the web this option uses the aria-live level to set the assertiveness
+  /// of the announcement. On iOS, Android, Windows, Linux, macOS, and Fuchsia
+  /// this option currently has no effect.
+  final Assertiveness assertiveness;
+
   @override
   Map<String, dynamic> getDataMap() {
-    return <String, dynamic>{
+    final Map<String, dynamic> dataMap = <String, dynamic> {
       'message': message,
       'textDirection': textDirection.index,
     };
+
+    if (assertiveness == Assertiveness.assertive) {
```

Comment:
Overall, this is a pretty strange design. Can we just fix up the failing tests in google?

Reply:
This PR is accompanied by an engine change as well, so my recommendation is to avoid touching too many repositories at once, which is achieved by making it non-breaking.

Having said that, fixing downstream tests sounds like a good idea. We can do it in multiple phases. Phase one - this change - introduce the API in a non-breaking fashion and stabilize it. Phase two - update tests to accept `assertiveness: 0`. Phase three - remove the condition.

Reply:
To be very clear: I don't think it is great practice that this was submitted while we were still discussing the best course forward. There should have at the very least been a TODO left here to remind future readers why we are (temporarily) doing this weirdness. Also, please clean this up ASAP once this has rolled into google.